parakeet_rocm/models/parakeet.py

```
@@ -92,6 +105,29 @@ def _get_cached_model(model_name: str = PARAKEET_MODEL_NAME) -> ASRModel:
+    Parameters:
+        model_name (str): Model name to look up in the cache.
+
+    Returns:
+        ASRModel | None: The cached model, or ``None`` if not present.
+    """
+    # lru_cache stores results in an internal dict-like .__wrapped__ closure.
+    # Accessing cache_parameters / cache_info does not trigger a load.
+    # The safest way to peek is to check the underlying OrderedDict.
+    try:
+        cache_dict = _get_cached_model.cache_parameters()  # type: ignore[attr-defined]
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)